pandas/tests/indexing/test_timedelta.py

```diff
@@ -13,7 +13,24 @@ def test_boolean_indexing(self):
                          [0, 1, 2, 10, 4, 5, 6, 7, 8, 9],
                          [10, 10, 10, 3, 4, 5, 6, 7, 8, 9]]
         for cond, data in zip(conditions, expected_data):
-            result = df.assign(x=df.mask(cond, 10).astype('int64'))
+            result = df.assign(x=df.mask(cond, 10).astype(df['x'].dtype))
```

Comment:
It still fails even with that attempted fix. Travis-CI builds is giving me error entirely unrelated to the change I have done.